engine/hud/ClickableHudElement.cpp

```diff
@@ -31,28 +32,16 @@ void ClickableHudElement::onInput(SDL_Event *event) {
         if (isCoordOnElement(event->button.x, event->button.y)) {
```

Comment:
Here the button is activated when mouse button is down, instead of when button is up.